Assets/MRTK/Providers/LeapMotion/LeapMotionDeviceManager.cs

```diff
@@ -105,6 +105,9 @@ public bool CheckCapability(MixedRealityCapability capability)
         /// </summary>
         private Vector3 leapHandsOffset => SettingsProfile.LeapControllerOffset;
 
+        // If the Leap Controller Orientation is the headset, then expose the settings for controller offset while using the LeapXRService Provider
```

Comment:
Super nit: The language here is a little confusing because "then expose" makes it feel like there's an action that happens nearby here (i.e. usually there's an if statement following this). Oddly enough this might be somewhere where having passive tone (i.e. settings for controller offset will be exposed in the inspector) would be clearer